e2e/cypress/integration/enterprise/ldap/ldap_test_setup.txt

```diff
@@ -0,0 +1,18 @@
+1. Provision the MM app in LDAP Provider following the steps described here: https://docs.mattermost.com/deployment/sso-ldap.html
+1.1 Add test data to LDAP Provider: https://github.com/mattermost/mattermost-server/blob/master/tests/test-data.ldif
```

Comment:
This is for information only, right? Since LDAP test-data is added via `make start-docker` - https://github.com/mattermost/mattermost-server/blob/master/Makefile#L136. Maybe mention it here?